modules/rgbd/include/opencv2/rgbd/volume.hpp

```diff
@@ -50,7 +50,7 @@ enum class VolumeType
     HASHTSDF = 1
 };
 
-cv::Ptr<Volume> makeVolume(VolumeType _volumeType, float _voxelSize, cv::Affine3f _pose,
+CV_EXPORTS_W cv::Ptr<Volume> makeVolume(VolumeType _volumeType, float _voxelSize, cv::Matx44f _pose,
                            float _raycastStepFactor, float _truncDist, int _maxWeight,
                            float _truncateThreshold, Point3i _resolution);
```

Comment:
Looks like `Point3i` causes compilation errors when generating Python bindings.
To resolve this, you may try to replace it to `Vec3i` type. Inside a function this can be converted to `Point3i` as it was done for `Affine3f`.

Reply:
done